api/api.py

```diff
@@ -468,12 +463,41 @@ def load(path: str, *args):
     load('./lib/Tiltfile', 'create_namespace')
     create_namespace('frontend')
 
+  A Tiltfile may only be executed once. If a Tiltfile is loaded multiple times,
+  the second load will use the results of the last execution.
+
   If ``path`` starts with ``"ext://"`` the path will be treated as a `Tilt Extension <extensions.html>`_.
 
   Example ::
 
     load('ext://hello_world', 'hi') # Resolves to https://github.com/tilt-dev/tilt-extensions/blob/master/hello_world/Tiltfile
     hi() # prints "Hello world!"
+
+  Note that ``load()`` is a language built-in. Read the
+  `specification <https://github.com/google/starlark-go/blob/master/doc/spec.md#load-statements>`_
+  for its complete syntax.
+
+  Because ``load()`` is analyzed at compile-time, the first argument MUST be a string literal.
+  """
+
+def load_dynamic(path: str) -> Dict[str, Any]:
+  """Load variables from another Tiltfile, and import them into the current scope.
+
+  Used when you want to define common functions or constants
+  to share across Tiltfiles.
+
+  Example ::
+
+    symbols = load_dynamic('./lib/Tiltfile')
+    create_namespace = symbols['create_namespace']
+    create_namespace('frontend')
+
+  Like :meth:`load`, each Tiltfile can only be executed once. Can also be used to load a `Tilt Extension <extensions.html>`_.
```

Comment:
```suggestion
  Like :meth:`load`, each Tiltfile will only be executed once. Can also be used to load a `Tilt Extension <extensions.html>`_.
```

nit - "can" makes me think that a repeated load is an error